en/docs/develop-components/develop-components-with-git.md

```diff
@@ -0,0 +1,20 @@
+# Develop Components With Git
+
+Choreo enables you to develop components by connecting your GitHub repository. You have the flexibility to either connect an existing repository and use your current source code to create the component, or start with an empty repository and develop the source code later. By integrating your Git repositories with Choreo, you can automate tasks and optimize workflows across multiple systems, all within the Choreo platform. As of now, Choreo only supports GitHub as its Git provider. 
+
+In Choreo, you can connect a Git repository that contains Ballerina source code or a Docker project. To connect your Git repository to Choreo as a Docker project, your Git repository should contain the Dockerfile and the build's context. The Dockerfile specifies the instructions for building the Docker image, while the build context is a set of files located in the specified path used to build the image.
```

Comment:
```suggestion
In Choreo, you can connect a Git repository that contains Ballerina source code or a Docker project. For Docker project, your Git repository should contain the Dockerfile and the build's context. The Dockerfile specifies the instructions for building the Docker image, while the build context is a set of files located in the specified path used to build the image.
```

Reply:
The start of the sentence is not correct and lacks context. Hence we need to have a continuation for better clarity.